detekt-rules-style/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/style/ForbiddenMethodCallSpec.kt

```diff
@@ -754,13 +754,27 @@ class ForbiddenMethodCallSpec(val env: KotlinCoreEnvironment) {
             }
 
             @Test
-            fun `should report nothing when using BigDecimal string constructor`() {
+            fun `should report when using BigDecimal string constructor`() {
```

Comment:
`not` is still correct here.

```suggestion
            fun `should not report when using BigDecimal string constructor`() {
```